src/Service/AuthorizationService.php

```diff
@@ -117,33 +117,33 @@ public function isGranted($identity, $permission, $context = null)
         }
 
         if ($this->hasAssertion($permission)) {
-            return $this->assert($this->assertions[(string) $permission], $identity, $context);
+            return $this->assert($this->assertions[(string) $permission], $permission, $identity, $context);
```

Comment:
Should you cast to string as well as you're saying that in the docblock? Otherwise you may receive an AssertionInterface.